src/components/Link/Link.stories.tsx

```diff
@@ -26,7 +26,7 @@ type Story = StoryObj<typeof Link>;
 export const Default: Story = {};
 
 /**
- * When using context, you can specify a trailing icon for the link.
+ * When using standalone context, you can specify a trailing icon for the link.
```

Comment:
What's standalone context, and/or can we clarify that?

Although I'm only looking at the diff, not the storybook. Might be more clear